.github/workflows/run_sqlfluff.yaml

```diff
@@ -0,0 +1,31 @@
+name: Run SQLFluff
+
+on:
+    pull_request:
+
+jobs:
+    sqlfluff_lint_check:
+        name: SQLFluff lint check
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+            - name: Get all changed sql files
+              id: changed-sql-files
+              uses: tj-actions/changed-files@v45
+              with:
+                files: dbt-cta/**.sql
+            - name: Setup python
+              uses: actions/setup-python@v5
+              with:
+                python-version: '3.12'
+            - name: Install SQLFluff
+              run: python3.12 -m pip install sqlfluff sqlfluff-templater-dbt
```

Comment:
@jitoquinto hmmm it should still pull in those config files since the sqlfluff command is running from the project directory (and presumably those get automatically pulled in by sqlfluff)